ci/eval/compare/utils.nix

```diff
@@ -52,12 +56,12 @@ rec {
       ]
     into
       [
-        { name = "hello"; platform = "aarch64-linux"; }
-        { name = "hello"; platform = "x86_64-linux"; }
-        { name = "hello"; platform = "aarch64-darwin"; }
-        { name = "hello"; platform = "x86_64-darwin"; }
-        { name = "bye"; platform = "aarch64-darwin"; }
-        { name = "bye"; platform = "x86_64-darwin"; }
+        { name = "hello"; platform = "aarch64-linux"; packagePath = [ ... ]; }
```

Comment:
This also needs to be changed in the documentation of `groupByPlatform` and `groupByKernel`.
Maybe you can just add `...` instead:
`{ name = "hello"; platform = "aarch64-linux"; }` --> `{ name = "hello"; platform = "aarch64-linux"; ... }`